settings/settings.go

```diff
@@ -29,13 +34,14 @@ func InitSettings() {
 	SMTPPassword = os.Getenv("SMTP_PASSWORD")
 
 	FromEmail = os.Getenv("FROM_EMAIL")
+	return nil
 }
 
 var EMAIL_INTERVAL time.Duration = 24 * time.Hour
```

Comment:
Don't name your constants like that, please. Official Go doc [suggests](https://go.dev/doc/effective_go#mixed-caps) using MixedCaps/mixedCaps for naming, even for constants.

Google go's style guide also suggests omitting UNDER_SCORES for constants
https://google.github.io/styleguide/go/decisions